test/Common/AutoFixture/Attributes/CustomAutoDataAttribute.cs

```diff
@@ -11,7 +11,7 @@ public CustomAutoDataAttribute(params Type[] iCustomizationTypes) : this(iCustom
 
     public CustomAutoDataAttribute(params ICustomization[] customizations) : base(() =>
     {
-        var fixture = new Fixture();
+        var fixture = new Fixture().WithAutoNSubstitutions();
```

Comment:
It was necessary before, some integration tests are broken on `main` (ie: GetBySsoUserAsync_Works_DataMatches).

Reply:
An `ILogger<T>` was introduced to a few repositories and since then the integration tests stopped working:

> AutoFixture was unable to create an instance from Microsoft.Extensions.Logging.ILogger`1[Bit.Infrastructure.Dapper.Repositories.OrganizationRepository] because it's an interface. There's no single, most appropriate way to create an object implementing the interface, but you can help AutoFixture figure it out.